src/rsa.rs

```diff
@@ -127,6 +127,30 @@ fn verify(params: &RSAParameters, msg: untrusted::Input,
 }
 
 
+pub mod primitive {
+    use untrusted;
+    /// Lower-level verification function for RSA signatures.
+    ///
+    /// If you do not have the public key available as DER-encoded
```

Comment:
I suggest something like "When the public key is in DER-encoded PKCS#1 ASN.1 format, it is recommended to use `ring::signature::verify()` with `ring::signature::RSA_PKCS1_*`, because then `ring::signature::verify()` will handle the parsing."

Then, s/If you do not have the public key available as DER-encoded PKCS#1-formatted ASN.1,/Otherwise/